joss/paper.md

```diff
@@ -58,22 +58,25 @@ are generally organized in our companion package, `xskillscore`.
 # Statement of Need
 While other climate verification packages exist (_e.g._, `s2dverification`
 [@Manubens:2018] written in R and `MurCSS` [@Illing:2014] written with python-based
-`CDO`-bindings [@CDO]), `climpred` is unique for many reasons. (1) It spans broad
-temporal scales of prediction, supporting the weather, subseasonal-to-seasonal (S2S),
-and seasonal-to-decadal (S2D) communities. (2) `climpred` supports `dask`
-[@dask; @Rocklin:2015] and thus works across all computational scales, from personal
-laptops to supercomputers (HPC). This leads to
-verification of a global 5° x 5° resolution climate prediction in a few seconds,
-compared to the 8 minutes required by `MurCSS`. This allows for a truly interactive
-analysis experience. However, note that benchmarking is inherently biased and `MurCSS`
-is valuable for their rigorous replication of decadal climate prediction metrics.
-(3) `climpred` is highly modular and supports the research process from end-to-end,
+`CDO`-bindings [@CDO]), `climpred` is unique for many reasons.
+(1) It spans broad temporal scales of prediction, supporting the weather,
+subseasonal-to-seasonal (S2S), and seasonal-to-decadal (S2D) communities.
+(2) `climpred` is highly modular and supports the research process from end-to-end,
 from loading in model output, to interactive pre-processing and analysis, to
-visualization. (4) `climpred` is part of the wider scientific python community, `pangeo`
+visualization.
+(3) `climpred` supports `dask` [@dask; @Rocklin:2015] and thus works across all
+computational scales, from personal laptops to supercomputers (HPC).
+(4) Flexibility and scaling leads to verification of a global 5° x 5° resolution of 
+already preprocessed climate predictions in 8 seconds, compared to the 8 minutes
```

Comment:
I would like to name numbers here. there for 8s vs 8min which we have references for. we now make the circumstances of these numbers. But I clearly want to share the "interactive" capabilities of climpred, which is our design strength

Reply:
I like the way you changed this up! Here is a revised suggestion to help it flow a little bit more. 

"Flexibility and scaling leads to verification of global 5° x 5° resolution of climate predictions in 8 seconds, compared to the 8 minutes required by `MurCSS`. However, note that these timing disparities are due to the fact that `MurCSS` has a more rigid framework that includes a pre-processing pipeline on all function calls. On the other hand, `climpred` is built to be modular, and thus pre-processing happens prior to verification calls. This allows for verification function calls to execute on the time scale of seconds, allowing for a truly interactive analysis experience."

I think we can drop the "benchmarking is inherently biased" part since that's sort of implied by our discussion here.

Reply:
I remembered we didnt have many spare words...

Reply:
"The paper should be between 250-1000 words."

Reply:
your last counting was <!-- 998 words total --> so we cannot extend.

Reply:
can you maybe try to suggest changes like I did below?